### ISSUE TITLE
Component only returning 1 item in array

### ISSUE THREAD

**Issue by chrisrobbins** [gitboss2000/react-native-swipeable-flat-list#4]
When I pass in the item, the output is only one item in the array. When I switch back to FLatList, it shows all of my items correctly. Is there a fix for this?

**Comment by gitboss2000** [comment]
I can't really answer this without the code producing the error, since it is working for me.

**Comment by mordaha** [comment]
The height of previous swipeable list items somehow collapsed to 0 and only last item shown

![](https://i.imgur.com/xBlN15n.png)

if you set height manually [here](https://github.com/gitboss2000/react-native-swipeable-flat-list/blob/master/src/components/SwipeableListItem.js#L309) all list items renders with this height

**Comment by gitboss2000** [comment]
Are you using the SwipeableFlatList under iOS? Today, I realized that components appear to not stretch to their children's dimensions on iOS, which they do on Android.
Anyways, I changed the SwipeableListItem to set the height according to the style.height of the item component as mordaha pointed out.
I hope this fixes your problems.

I also added an example and still liked to get some information on your plattform and the code you used.
